pepper-apis/src/test/java/org/broadinstitute/ddp/route/GetActivityInstanceRouteTest.java

```diff
@@ -155,7 +155,7 @@ private static void setupActivityAndInstance(Handle handle) throws Exception {
         txt2 = TextQuestionDef.builder(TextInputType.TEXT, "TEXT_DRUG", newTemplate())
                 .setSuggestionType(SuggestionType.DRUG)
                 .build();
-        Template txt3Tmpl = Template.html("$foo $DDP_PARTICIPANT_FIRST_NAME's favorite color?");
+        Template txt3Tmpl = Template.html("$foo $ddp.participantFirstName()'s favorite color?");
```

Comment:
Need to match method name exactly. Can also use the other curly syntax: `${ddp.participantFirstName()}`.